test/e2e/helpers.js

```diff
@@ -528,7 +528,7 @@ const onboardingCompleteWalletCreationWithOptOut = async (driver) => {
   await Promise.all(
     (
       await driver.findClickableElements(
-        '.toggle-button.toggle-button--on:not([data-testid="basic-functionality-toggle"] .toggle-button)',
+        '.toggle-button.toggle-button--on:not([data-testid="basic-functionality-toggle"] .toggle-button):not([data-testid="profile-sync-toggle"] .toggle-button)',
```

Comment:
why is this change needed?

Reply:
In the driver actions above (a few lines above), we have already toggled the profile-sync button off, so we don't want to turn it back on.

NOTE - we will discuss with the team to better align with the global "Basic Functionality" toggle. (just some thoughts), but if the Basic Functionality is turned off, do we want to disable or hide our notification toggles & features. May be worth discussing with other teams on how this is solved.

Reply:
oh, I see. I guess we could also just delete the line above?

Reply:
Since our toggle opens a modal to warn you that you are turning off, I think the lines above are valid.

The snippet above, it toggles profile sync off, and then confirms turning off profile sync.
```js
  await driver.clickElement(
    '[data-testid="profile-sync-toggle"] .toggle-button',
  );
  await driver.clickElement('[data-testid="submit-button"]');
```

Reply:
I do get it though, this is some crazy CSS selectors here. I'll see if we can filter or do something else instead.

Reply:
oh, sorry, I missed that part about the modal. We can leave this as is for now and clean up later